### PR TITLE
Requesting a pull to mozilla:v0.5.x from k88hudson:t1682

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/f3dff3cb5d4c4ae/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/f3dff3cb5d4c4ae/output.txt

Total script time: 0.47 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/34d14ef5e2b2e06/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/34d14ef5e2b2e06/output.txt

Total script time: 0.80 mins

Server is running on http://pr610.butyr.org/templates/test.html

File: templates/supported/newscaster/index.html [hunk 1]
```diff
@@ -34,27 +34,27 @@
 <body>
 <div class="wrapper">
   <header>
-    <h2 class="logo">knn report</h2>
+    <h2 class="logo"></h2>
```
Comment (cadecairos) [inline templates/supported/newscaster/index.html:37]: How come this title was removed completely?

Comment (k88hudson) [inline templates/supported/newscaster/index.html:37]: Well, since we can't edit it yet, it's not really useful for anything, so better left out.

**Comment by jbuck** [comment]
Staged on master: https://github.com/mozilla/butter/commit/7b34136005089fd0e710a4ad3358b29088e88a3a

**Pull request closed without merging**